contracts/AssetToken.sol

```diff
@@ -75,14 +85,29 @@ contract AssetToken is ERC223Interface, ERC20Interface {
         return _balances[owner];
     }
 
-    function fund(address member, uint256 value) public onlyOwner noOwnerAsCounterparty(member) {
+    function switchTrading() public onlyOwner {
```

Comment:
I would have made it explicit which state you were setting. something like `emergencyStop(bool enable); `